### DISCUSSION TITLE
How much network traffic is sent out by the runners?

### DISCUSSION THREAD

**Discussion by yuvalshi0** [actions/runner#2410] [General]
Hi,
We recently started hosting our runners using the[ actions-runner-controller](https://github.com/actions/actions-runner-controller) on our personal EKS, after a few days, we started seeing a high NAT bill (which is priced by the amount of data sent our through the Nat), I wanted to know, how much traffic does the runner trasmit?  GB's? MB's?

**Comment by TingluoHuang** [comment]
it depends on what you do within your workflow job, the runner itself should not consume much network traffic.
- Do you upload lots of artifacts during your workflow?
- Do we have lots of log content generated as part of your job?
- Any other outbound communication your job tries to do?

**Reply by yuvalshi0** [reply]
We run a python job, about ~10 minutes, 10 runners , very verbose (log wise, we print the test stdout), we do use the [setup-python](https://github.com/actions/setup-python) caching, which probably uploads a large artifact (?), I see we output about ~2GB per run, does that sounds plausible ?

**Comment by pratikvasa** [comment]
we also host action runners on EKS. We isolated and checked network throughput. I ran 3 commands, 
```
npm ci
npx playwright install chromium --with-deps
npx playwright install chrome
```

just running these three commands saw a 1.8GB receive and transmit on the runner.
I could not understand why runners would need to transmit so much data
